### PR TITLE
Parse representatives sheet in The Trexler File

### PR DESCRIPTION
**Pull request by holdenhinkle**
References https://github.com/department-of-veterans-affairs/va.gov-team/issues/73253

## Summary

- Parse the `Representatives` sheet in Trexler File to support VSOfficers
- Refactor `RepAddresses::QueueAddressUpdates` and move all XLSX file processing into its own class `RepAddresses::XlsxFileProcessor`
- Update tests
- Write new tests

## Related issue(s)

- [department-of-veterans-affairs/va.gov-team#0000
- *Link to ticket created in va.gov-team repo*
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73253)

## Testing done
- Updated tests
- Wrote new tests

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
The Find a Rep product which is not live yet (it's behind a feature flag)

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: modules/veteran/app/sidekiq/rep_addresses/xlsx_file_processor.rb [hunk 1]
```diff
@@ -0,0 +1,162 @@
+# frozen_string_literal: true
+
+module RepAddresses
+  class XlsxFileProcessor
+    include SentryLogging
+
+    SHEETS_TO_PROCESS = %w[Attorneys Representatives].freeze
```
Comment (jvcAdHoc) [inline modules/veteran/app/sidekiq/rep_addresses/xlsx_file_processor.rb:7]: Based on Martin's most recent email, we probably want to keep agents in here

File: modules/veteran/spec/sidekiq/rep_addresses/queue_address_updates_spec.rb [hunk 2]
```diff
@@ -4,74 +4,57 @@
 require 'sidekiq/testing'
 Sidekiq::Testing.fake!
 
-# Test suite for RepAddresses::QueueAddressUpdates.
-# Tests focus on both the observable outcomes of methods and the direct testing of Sentry logging.
 RSpec.describe RepAddresses::QueueAddressUpdates, type: :job do
+  describe 'modules and initialization' do
```
Comment (jvcAdHoc) [inline modules/veteran/spec/sidekiq/rep_addresses/queue_address_updates_spec.rb:8]: for symmetry, should `XlsxFileProcessor` require Roo and its spec have a similar assertion?

**Review by cosu419 — APPROVED** [review]
LGTM

File: modules/veteran/app/sidekiq/rep_addresses/xlsx_file_processor.rb (hunk 1 above)
Comment (holdenhinkle) [reply modules/veteran/app/sidekiq/rep_addresses/xlsx_file_processor.rb:7]: I'll add agents in a separate PR if we need to.

File: modules/veteran/spec/sidekiq/rep_addresses/queue_address_updates_spec.rb (hunk 2 above)
Comment (holdenhinkle) [reply modules/veteran/spec/sidekiq/rep_addresses/queue_address_updates_spec.rb:8]: Roo isn't required. But I added a test to make sure SentryLogging is required.

**Pull request merged**